app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -17,6 +18,44 @@
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs .govuk-breadcrumbs__list-item {
+  margin-left: 0;
+  padding-left: 0;
+  margin-right: 20px;
+  @include chevron($direction: "right", $color: govuk-colour("dark-grey"));
+
+  &:before,
+  &:last-of-type:after {
+    display: none;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    display: none;
```

Comment:
Not that I want to throw another spanner in the works, but it's just occurred to me that the change being proposed here reduces the functionality of the breadcrumb to only a link to the homepage, whereas previously it provides both a visual representation of where the user is and navigation options for moving 'up'. 

Has there been any user research into this? I'm concerned that this reduction, while making better use of the vertical space, reduces the usefulness of the breadcrumb significantly. One alternative would be to do this change as proposed, but when the user clicks on any part of the breadcrumb, to show the entire thing?

Reply:
@andysellick I think the commit/PR name is a bit misleading - it's actually showing the Home and parent page on mobile, rather than the home and current page. Is that correct @RobBond-GDS ?

Reply:
That would be better, yes, if that's the case then the PR description needs to be updated as well, along with the documentation for this component.

Reply:
Yes it is home and parent page, I have updated the PR description and included the snippet for the research and reasoning as mentioned in the design doc on the trello card